openmp/runtime/src/z_Linux_asm.S

```diff
@@ -17,6 +17,27 @@
 
 #include "kmp_config.h"
 
+#if KMP_OS_LINUX
```

Comment:
Maybe this section could be moved into the 
`#if (KMP_OS_LINUX || KMP_OS_DARWIN || KMP_OS_WINDOWS) && (KMP_ARCH_AARCH64 || KMP_ARCH_AARCH64_32 || KMP_ARCH_ARM)` protected section.

```suggestion
#if KMP_OS_LINUX && (KMP_ARCH_AARCH64 || KMP_ARCH_AARCH64_32)
```
as it contains AARCH64 specific values.

Reply:
I've just added a new commit that moves the code as suggested.